source/Applications/Advanced/HandEyeCalibration/HandEyeCalibration/HandEyeCalibration.cpp

```diff
@@ -70,6 +70,26 @@ namespace
         const auto settings = Zivid::CaptureAssistant::suggestSettings(camera, parameters);
         return camera.capture(settings);
     }
+
+    Zivid::Calibration::HandEyeOutput performCalibration(const std::vector<Zivid::Calibration::HandEyeInput> &input)
+    {
+        while(true)
+        {
+            std::cout << "Enter type of calibration, eth (for eye-to-hand) or eih (eye-in-hand): ";
```

Comment:
std::cout << "Enter type of calibration, eth (for eye-to-hand) or eih (**for** eye-in-hand):